easy_my_coop_payment_term/views/res_company_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_company_easy_my_coop_loan">
+        <field name="name">res.company.form.easymy.coop</field>
+        <field name="inherit_id" ref="easy_my_coop.view_company_inherit_form2" />
+        <field name="model">res.company</field>
+        <field name="arch" type="xml">
+            <field name="subscription_maximum_amount" position="after">
+                 <field name="default_payment_term" />
+            </field>
```

Comment:
Should this field be invisible under certain conditions?

Reply:
I don't think so, this is a dedicated module for payment term, so it is installed only if the user wants to defines a default payment term.

Reply:
I'm sorry, I should have elaborated. Should this field be invisible for defined companies that are not cooperatives, or defined companies that are not 'Your Company'? @robinkeunen probably knows a good answer to this.

Reply:
OK I see your point :) I agree that it ideally it should not appear on all companies, only for those who will have subscription requests, but I don't know how to get this information...

******Your** point also makes me realize that the field 'default payment term' is not precise enough, it should be 'default payment term for subscriptions request', or something else. @robinkeunen do you have a suggestion ?

Reply:
I think you're confusing `res.company` with a `res.partner` which is a company (`res_partner.is_company == True`). 

`res.company` represents "Your Company", the company that is using odoo. That's where the configurations sit and therefore, I don't think `default_payment_term` should be invisible

Reply:
And indeed, `default_subscription_request_payment_term` is probably better even if a bit long.

Reply:
Regarding the task, there were some discussions with Catherine (and some inputs from you too) and this is what I understood. Catherine will tell me if it is not right.

Reply:
Actually I am not sure about changing the name, since default payment term is in the Easy_my_coop menu, we know it refers to subscription requests no ?

Reply:
For the UX name, it's maybe clear enough but not when we'll be using it un the code.

Reply:
It's done